src/main/java/org/opensearch/security/dlic/rest/api/NodesDnApiAction.java

```diff
@@ -52,10 +53,10 @@
  * By default the APIs are disabled and can be enabled by a YML setting - {@link ConfigConstants#OPENDISTRO_SECURITY_NODES_DN_DYNAMIC_CONFIG_ENABLED}
  *
  * The backing data is stored in {@link ConfigConstants#OPENDISTRO_SECURITY_CONFIG_INDEX_NAME} which is populated during bootstrap.
- * For existing clusters, {@link com.amazon.opendistroforelasticsearch.security.tools.OpenDistroSecurityAdmin} tool can
+ * For existing clusters, {@link SecurityAdmin} tool can
  * be used to populate the index.
  *
- * See {@link com.amazon.opendistroforelasticsearch.security.dlic.rest.api.NodesDnApiTest} for usage examples.
+ * See {@link org.opensearch.security.dlic.rest.api.NodesDnApiTest} for usage examples.
```

Comment:
Should we give the full class name or not? line 56, 59 is different.

Reply:
We actually don't need the full path. I've changed the line 56.